the-collab-lab/tcl-38-smart-shopping-list
# Final stylings

## Description

This PR shows the final styling updates for version 1 of the TCL-38 Smart Shopping List.

## Related Issue

Closes #13. 

## Acceptance Criteria

Your team will self-organize around a lightweight design process to come up with a polished look-and-feel for your app.
This will require you to make decisions together as a whole team, then implement your vision. The process outlined here is our best suggestion for how to proceed, but you are free to take whatever approach you see fit!

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

| home page  | add item page | item list page |
| ------------- | ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/40015502/156910445-fc51cf39-7545-49a2-864f-a4507f3a8d55.png)|![image](https://user-images.githubusercontent.com/40015502/156910454-e1faf5f9-dd73-44f9-9456-53c3db401b4c.png)|![image](https://user-images.githubusercontent.com/40015502/156910458-b20d7e18-cb57-4a9b-bc15-29357b5f0a25.png)

## Testing Steps / QA Criteria

Pull down the branch, run npm install, and test all features, with and without a token, check responsiveness for mobile and desktop.
